base/Enums.jl

```diff
@@ -29,7 +29,7 @@ end
 """
     @enum EnumName EnumValue1[=x] EnumValue2[=y]
 
-Create an [`Enum`](:obj:`Enum`) type with name `EnumName` and enum member values of
+Create an `Enum` type with name `EnumName` and enum member values of
```

Comment:
Those ones as well. What I'll do is put together a list of which ones are missing after this conversion is done and open an up-for-grabs issue to track which ones still need documenting.